cmake/projects/Qt/schemes/url_sha1_qt.cmake.in

```diff
@@ -215,6 +216,10 @@ if(BUILD_SHARED_LIBS)
   list(APPEND configure_opts "-shared")
 else()
   list(APPEND configure_opts "-static")
+  hunter_check_toolchain_definition(NAME "_MT" DEFINED _hunter_mt)
```

Comment:
`_MT` defined by both `/MT` and `/MD`. As far as I understand we need to check `_DLL`:
* https://github.com/ruslo/hunter/issues/1190#issuecomment-343739373
* https://github.com/ruslo/hunter/pull/1203#issuecomment-344561420

Reply:
you right, i have to check both toolchain definitions.